Tests/LibWeb/Text/input/HTML/requestAnimationFrame-gc.html

```diff
@@ -0,0 +1,14 @@
+<script src="../include.js"></script>
+<script>
+    asyncTest((done) => {
+        requestAnimationFrame(() => {
+            println("Collect garbage");
+            internals.gc();
```

Comment:
I wonder if it makes any sense for us to have a test configuration that runs GC on every possible point that it may execute?

Reply:
That's what `js -g` does but uh.. I hope you have a couple days if you want to run the web tests in that mode 😅